core/src/main/java/org/hisp/dhis/android/core/tracker/exporter/TrackerDownloadCall.kt

```diff
@@ -66,7 +66,7 @@ internal abstract class TrackerDownloadCall<T, Q : BaseTrackerQueryBundle>(
             send(progressManager.increaseProgress(TrackedEntityInstance::class.java, true))
         } else {
             val relatives = RelationshipItemRelatives()
-            systemInfoModuleDownloader.downloadWithProgressManager(progressManager).blockingSubscribe()
+            systemInfoModuleDownloader.downloadWithProgressManager(progressManager)
```

Comment:
As the systemDownloader returns a suspend function instead of a Single, the TrackerDownloadCall can be simplified @andresmr